src/App.vue

```diff
@@ -172,6 +173,8 @@ import PlioAPIService from "@/services/API/Plio.js";
 import { mapActions, mapState, mapGetters } from "vuex";
 import { useToast } from "vue-toastification";
 
+const MOBILE_SCREEN_WIDTH_THRESHOLD = 500;
```

Comment:
- Make the other value in `PlioListItem.vue` as TAB_SCREEN_WIDTH_THRESHOLD
- Also, make these constants and methods (`isMobileScreen` and `isTabScreen`) as methods imported from the Utilities module. And those modules will use the store value in turn.